plugins/rtmp-services/data/services.json

```diff
@@ -2193,6 +2193,37 @@
                 "max audio bitrate": 320,
                 "x264opts": "scenecut=0"
             }
+        },
+        {
+            "name": "Volume",
+	    "alt_names": ["Volume.com"],
```

Comment:
This is only required when a service is being migrated, as this is a new service this line does not need to be here (also the indentation is wrong).